PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py

```diff
@@ -1,13 +1,37 @@
 import FWCore.ParameterSet.Config as cms
+from TrackingTools.TrackAssociator.default_cfi import TrackAssociatorParameterBlock
+
+TrackAssociatorParameterBlock.TrackAssociatorParameters.useMuon = cms.bool(False)
+TrackAssociatorParameterBlock.TrackAssociatorParameters.useCalo = cms.bool(False)
+TrackAssociatorParameterBlock.TrackAssociatorParameters.useHO = cms.bool(False)
+TrackAssociatorParameterBlock.TrackAssociatorParameters.usePreshower = cms.bool(False)
+TrackAssociatorParameterBlock.TrackAssociatorParameters.EERecHitCollectionLabel = cms.InputTag("reducedEcalRecHitsEE")
+TrackAssociatorParameterBlock.TrackAssociatorParameters.EBRecHitCollectionLabel = cms.InputTag("reducedEcalRecHitsEB")
+TrackAssociatorParameterBlock.TrackAssociatorParameters.HBHERecHitCollectionLabel = cms.InputTag("reducedHcalRecHits","hbhereco")
+TrackAssociatorParameterBlock.TrackAssociatorParameters.HORecHitCollectionLabel = cms.InputTag("reducedHcalRecHits","horeco")
```

Comment:
this globally modifies TrackAssociatorParameterBlock, which is bad.
Please clone and modify the copy.

This should fix problems running tests in jenkins.